app/Testing/Context/MailSending.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Brewmap\Testing\Context;
+
+use Behat\Behat\Context\Context;
+use Brewmap\Notifications\EmailChangeNotification;
+use Illuminate\Notifications\AnonymousNotifiable;
+use Illuminate\Support\Facades\Notification;
+
+class MailSending implements Context
+{
+    protected $signedUrl;
+
+    public function __construct()
+    {
+        Notification::fake();
```

Comment:
I don't like this solution. I would prefer mocking it by standard mock service, but we can wait for someone else thoughts and comments.

Reply:
Facade is using `Illuminate\Support\Testing\Fakes\NotificationFake`, maybe we can use it here directly?

Reply:
I am trying use `Illuminate\Support\Testing\Fakes\NotificationFake` but when I checked assertion, function threw error...

Reply:
Could you be more pricise? What kind of error? Maybe you could paste error message here?